text/0000-add-peers-in-runtime.md

```diff
@@ -0,0 +1,222 @@
+- Feature Name: `add_peers_in_runtime`
+- Start Date: 2018-11-12
+- RFC PR: (leave this empty)
+- Sawtooth Issue: (leave this empty)
+
+# Summary
+[summary]: #summary This RFC proposes to implement the possibility to add new
+peers and remove the existing connections in runtime (through the component
```

Comment:
Recommend: "This RFC proposes functionality to add and remove static peer connections while a validator node is running."